src/pages/Lending/Pool/components/Borrow/BorrowConfirm.tsx

```diff
@@ -108,6 +115,8 @@ export const BorrowConfirm = ({
     select: mutation => mutation.state.status,
   })
 
+  console.log({ lendingMutationStatus })
```

Comment:
revert me, sanity checking that status detection was still happy with the `mutateAsync` sig change

Reply:
8e1e54bcfd